.github/workflows/tests_windows.yml

```diff
@@ -119,7 +119,9 @@ jobs:
 
       - name: Create installation directory
         if: steps.kokkos-cache.outputs.cache-hit != 'true'
-        run: mkdir -p D:\a\install_dir\${{ matrix.exec_model }}
+        run: |
+          rm D:\a\install_dir\${{ matrix.exec_model }} -r -fo
```

Comment:
The CI was [complaining](https://github.com/PennyLaneAI/pennylane-lightning/actions/runs/4755714887/jobs/8450279550#step:5:5) the directory already exists. I think `mkdir -p` is not a valid powershell option, or doesn't work as on Linux. This usually does not show up because this step is skipped after the cache hit.

Reply:
Makes sense. Thanks for clarifying.